src/interface/TWAnyAddress.cpp

```diff
@@ -16,6 +16,10 @@ using namespace TW;
 struct TWAnyAddress {
     TWString* address;
     enum TWCoinType coin;
+
+    ~TWAnyAddress() {
+        TWStringDelete(address);
+    }
```

Comment:
Using `TWString*` here was not a good idea, let's follow `TWPublicKey` implementation, Add a c++ class `AnyAddress` with `std::string` and destructor

Reply:
I think if memory leaks are not the issue, we can leave it as is - no need to make it more complex.